### PR TITLE
Add support for orientdb_users resources key

### PR DESCRIPTION
**Pull request by ernestas-poskus**
@migibert 

Adding support for orientdb_users.resources hash key.

### CODE REVIEW

**Comment by migibert** [comment]
Hi,

Thanks for this PR, could you add a little word of documentation about this ?

File: defaults/main.yml [hunk 1]
```diff
@@ -17,8 +17,11 @@ orientdb_enable_http: True
 orientdb_http_port_range: 2480-2490
 orientdb_force_install: False
 orientdb_users:
-- name: root
-  password: root
+  - name: root
+    password: root
+  - name: guest
+    password: guest
+    resources: 'connect,server.listDatabases,server.dblist'
```
Comment (migibert) [inline defaults/main.yml]: What is the difference between server.listDatabases and server.dblist ?
Could you add some tests about resources ?

File: tests/example.yml [hunk 2]
```diff
@@ -18,8 +18,11 @@
     orientdb_enable_http: True
     orientdb_http_port_range: 2480-2490
     orientdb_users:
-    - name: root
-      password: root
+      - name: root
```
Comment (migibert) [inline tests/example.yml]: Could you add some tests about root's resources and gest's resources ?

File: defaults/main.yml (hunk 1 above)
Comment (ernestas-poskus) [inline defaults/main.yml]: I have copied from static guest user definition. From documentation was able to find 

```
server.listDatabases - Lists available databases on the server.
```

I guess ```server.dblist``` means ```list databases``` too but it is deprecated and left for compatibility reasons

File: tests/example.yml (hunk 2 above)
Comment (ernestas-poskus) [inline tests/example.yml]: @migibert sure, where ? circle.yml or serverspec ?

Comment (migibert) [inline tests/example.yml]: Ideally both, but at least in circle.yml. Thanks!

**Pull request merged**